From d375cfbcc61b7709696041bc5ed960cd2f9fbf03 Mon Sep 17 00:00:00 2001
From: Portisch <hugo.portisch@yahoo.de>
Date: Tue, 23 Dec 2025 14:39:47 +0100
Subject: [PATCH] Revert "terminal-util: during terminal reset clear from
 beginning of line to end of screen"

This reverts commit 4efd46c44bc599a71da1ff24149291ef10df61ff.
---
 src/basic/terminal-util.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/basic/terminal-util.c b/src/basic/terminal-util.c
index 9abef38aad..e2911e235d 100644
--- a/src/basic/terminal-util.c
+++ b/src/basic/terminal-util.c
@@ -970,9 +970,7 @@ int terminal_reset_ansi_seq(int fd) {
         k = loop_write_full(fd,
                             "\033[!p"      /* soft terminal reset */
                             "\033]104\007" /* reset colors */
-                            "\033[?7h"     /* enable line-wrapping */
-                            "\033[1G"      /* place cursor at beginning of current line */
-                            "\033[0J",     /* erase till end of screen */
+                            "\033[?7h",    /* enable line-wrapping */
                             SIZE_MAX,
                             100 * USEC_PER_MSEC);
         if (k < 0)
-- 
2.34.1

